src/library/handleHttpResponseError.js

```diff
@@ -45,9 +41,6 @@ const handleHttpResponseError = ({ error, callback, logoutMermaid }) => {
 
     return
   }
-
-  // Display a generic error if we haven't returned yet
-  toast.error(...getToastArguments(language.error.generic))
```

Comment:
We were being too liberal on the toasting of generic error messages. This http error handler utility is just used to intercept some http errors, if none are intercepted, it doesnt mean we should fallback within the utility to toast out some generic blabber (we always have the option to do that with the passed in callback)

Reply:
nice catch! agreed